src/index.css

```diff
@@ -1,27 +1,95 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+#root {
+  margin: 15% 35% 0 35%;
+}
+
+.calculator_container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: repeat(5, 2.5rem);
+  height: 15rem;
+}
+
+.display_results {
+  grid-area: 1 / 1 / 2 / 5;
+  width: 100%;
+  background-color: rgb(143, 159, 159);
+  outline: none;
+  border: 1px solid grey;
+  text-align: end;
+  font-size: 1rem;
+  color: #fff;
+  height: 2.5rem;
+}
+
+.complexed_opearations {
+  grid-area: 2 / 1 / 3 / 4;
+  display: flex;
+  background-color: rgb(206, 202, 202);
+  font-size: 0.7rem;
+}
+
+.complex_operations {
+  width: 33.33%;
+  border: 1px solid grey;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.simplest_operation {
+  grid-area: 2 / 4 / 7 / 5;
+}
+
+.sim_operations {
+  width: 100%;
+  height: 2.5rem;
+  border-top: 1px solid #d1cdcd;
+  background-color: rgb(239, 123, 28);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.num_pad {
+  grid-area: 3 / 1 / 7 / 4;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.num_pad p {
+  height: 2.5rem;
+  width: 100%;
+  background-color: rgb(206, 202, 202);
+  border: 1px solid grey;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#num_pad_zero {
+  grid-area: 6 / 1 / 7 / 3;
+}
+
+#num_pad_dot {
+  grid-area: 6 / 3 / 7 / 4;
+}
+
+#results {
+  justify-content: end;
+  color: white;
+  background-color: rgb(143, 159, 159);
+  border: none;
+  padding: 0;
+  text-align: right;
+  width: 100%;
+  height: 100%;
```

Comment:
- [ ] Good job so far 👏👏 However, as mentioned in the project requirements, kindly make sure that the calculator matches [the given design](https://raw.githubusercontent.com/microverseinc/curriculum-react-redux/main/math-magicians/images/calculator.png?token=GHSAT0AAAAAAB5V373U7ZO62ZDZO6QA4RXOZJRT5RA). For example, I noticed that some of the the operators currently being used are bit different from the given, also the numbers are not placed according to the given design ⛏Could you please fix this as per [this project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=a%20React%20component%20that%20matches%20this%20design%3A) Also I noticed that colors currently used are a bit different from the one that we have in the design guidelines. Kindly have a look at the comparison below 👇


Current design | Given design
---| ---|
![image](https://github.com/tatouroberto/maths-magicians/assets/106119105/64087438-fd51-462c-a0cf-3191f6dfa901) | ![image](https://github.com/tatouroberto/maths-magicians/assets/106119105/90656c3b-882a-4cc8-bfed-68b6b7623f5d)